src/util/settings/connection_client.cc

```diff
@@ -213,14 +214,16 @@ QString serverScriptDefault() {
   server_script = qApp->applicationDirPath() + "/../veles-server/srv.py";
 #elif defined(Q_OS_LINUX)
   server_script = qApp->applicationDirPath() + "/../share/veles-server/srv.py";
-#else
+#elif defined(Q_OS_MAC)
   server_script = qApp->applicationDirPath() + "/../Resources/veles-server/srv.py";
+#else
+  #error OS not supported
```

Comment:
But which path should we use for that 'other OS'? Before this change it would just pick the random one which happened to be in `#else`, which almost for sure won't work. This way someone compiling this for exotic OSes will explicitly have to fix the paths here.